compiler/luci-interpreter/src/kernels/Pow.test.cpp

```diff
@@ -96,6 +96,27 @@ TEST(PowTest, Input_Output_Type_NEG)
   EXPECT_ANY_THROW(kernel.configure());
 }
 
+TEST(PowTest, Input_Output_Type_NEG)
```

Comment:
(optional) To reduce changes, if not should add in alphabetical order, it would be better to add `Input_Type_Mismatch_NEG` after `Input_Output_Type_NEG`.

Reply:
Thank you for your advice, @seanshpark :)
I will edit and re-commit.